numba/cuda/tests/cudapy/test_debug.py

```diff
@@ -47,7 +48,11 @@ def check_debug_output(self, out, enabled_dumps):
                 self.assertRaises(AssertionError, check_meth, out)
 
     def _check_dump_bytecode(self, out):
-        self.assertIn('BINARY_ADD', out)
+        if PYVERSION == (3, 11):
```

Comment:
Are we expecting this to change back for 3.12? If not, perhaps
```suggestion
        if PYVERSION >= (3, 11):
```

will be more robust for the future?

Reply:
Suggesting leaving it as-is following OOB discussion that this is in line with other similar changes. If it breaks for 3.12 I'd like to change this test to compare against the bytecode from `dis.dis()` anyway so it's Python-version-invariant.

Reply:
From OOB discussion, electing to leave this as-is such that if it breaks again for 3.12 the test itself might need a change to the way in which it works.